docs/sources/enterprise/export-pdf.md

```diff
@@ -10,15 +10,10 @@ weight = 900
 
 # Export dashboard as PDF
 
-Export dashboard as PDF allows you to generate PDFs from any of your dashboards and save it to file.
+You can generate PDFs from any of your dashboards and save it to file.
 
 > Only available in Grafana Enterprise v6.7+.
 
-## Generating dashboard PDF
-
-Dashboard can be exported to PDF from **Share dashboard** menu.
-
-1. Click on the share icon in the dashboard menu.
-1. Go to PDF tab.
-1. Select the layout option for exported dashboard. By default it is set to **Portrait** but it's also possible to select **Landscape**.
+1. Click the title of the dashboard that you want to export as PDF and then click **Share**.
```

Comment:
That's not true it seems. Share modal available from share button, not dashboard title.
![Screenshot from 2020-03-13 12-51-00](https://user-images.githubusercontent.com/4932851/76609982-51e13680-6529-11ea-84b2-b58aa25b4015.png)